challenge-13/challenge-13.js

```diff
@@ -1,150 +1,180 @@
+
+  /*
+  Crie um array e mostre no console a representação em String desse array,
+  usando o método visto na aula 13.
+  */
+  console.log( 'O array em formato de string é:' );
+  var arr = [1, 2, 3, 4, 5];
+  console.log(arr.toString());
+
+  /*
+  Crie 2 arrays `sul` e `sudeste`, que serão as regiões do Brasil.
+  Cada array deve conter os estados dessa região.
+  */
+  var sul = ['Paraná', 'Santa Catarina', 'Rio Grande do Sul'];
+  var sudeste = ['São Paulo', 'Minas Gerais', 'Rio de Janeiro', 'Espírito Santo'];
+
+  /*
+  Crie uma variável chamada `brasil`, que irá receber as duas regiões
+  concatenadas. Mostre o `brasil` no console.
+  */
+  console.log( '\nAlguns Estados do Brasil:' );
+  var brasil = sul.concat(sudeste);
+  console.log(brasil);
+
+  /*
+  Adicione 3 novos estados da região Norte no início do array e mostre no console.
+  */
+  console.log( '\nMais estados adicionados:' );
+  brasil.unshift('Amazonas', 'Acre', 'Pará')
+  console.log(brasil);
+
+  /*
+  Remova o primeiro estado do array `brasil` e mostre-o no console.
+  */
+  console.log( '\nEstado removido:' );
+  console.log(brasil.shift());
+
+  /*
+  Crie um novo array chamado `newSul`, que receba somente os estados do sul,
+  pegando do array `brasil`. Não remova esses itens de `brasil`.
+  */
+  var newSul = brasil.slice(2, 5);
+
+  /*
+  Mostre no console os estados que estão em `newSul`.
+  */
+  console.log( '\nEstados do Sul do Brasil:' );
+  console.log(newSul);
+
+  /*
+  Mostre no console todos os estados que estão em `brasil`.
+  */
+  console.log( '\nAlguns Estados do Brasil:' );
+  console.log(brasil);
+
+  /*
+  Crie um novo array chamado `nordeste`, que tenha os estados do nordeste.
+  */
+  var nordeste = [
+    'Alagoas',
+    'Bahia',
+    'Ceará',
+    'Maranhão',
+    'Paraíba',
+    'Pernambuco',
+    'Piauí',
+    'Rio Grande do Norte',
+    'Sergipe'
+  ];
+
+  /*
+  Mostre no console os estados do nordeste.
+  */
+  console.log( '\nEstados do Nordeste:' );
+  console.log(nordeste);
+
+  /*
+  Remova de `brasil` os estados do `sudeste`, colocando-os em uma variável
+  chamada `newSudeste`.
+  */
+  var newSudeste = brasil.splice(5, 4);
+
+  /*
+  Adicione os estados do `nordeste` ao array `brasil`. Esses estados devem
+  ficar no mesmo nível que os estados já existentes, não em um array separado.
+  */
+  brasil = brasil.concat(nordeste);
+
+  /*
+  Mostre no console os estados em `newSudeste`.
+  */
+  console.log( '\nEstados em newSudeste:' );
+  console.log(newSudeste);
+
+  /*
+  Mostre no console os estados do `brasil`.
+  */
+  console.log( '\nAlguns estados do Brasil:' );
+  console.log(brasil);
+
+  /*
+  usando forEach, percorra o array `brasil` e gere um novo array chamado
+  `newBrasil`. Esse array deve ter cada item como um objeto, com as
+  propriedades:
+  - `id`: que será o índice do array `brasil`,
+  - `estado`: que será o estado do array `brasil`.
+  */
+  var newBrasil = [];
+  brasil.forEach(function(value, index, array){
```

Comment:
-1
Não é necessário adicionar todos os parâmetros. Só adicione os que você for utilizar.